scripts/context_retrieval/wrangle_llama_index_data.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    ].sample(n=1000)
</pre><p>1000 is a magic number - best to hoist and explain?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>